src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java

```diff
@@ -289,7 +290,7 @@ private void displaySlimefunItem(ChestMenu menu, ItemGroup itemGroup, Player p,
             menu.addItem(index, new CustomItemStack(ChestMenuUtils.getNoPermissionItem(), sfitem.getItemName(), message.toArray(new String[0])));
             menu.addMenuClickHandler(index, ChestMenuUtils.getEmptyClickHandler());
         } else if (isSurvivalMode() && research != null && !profile.hasUnlocked(research)) {
-            menu.addItem(index, new CustomItemStack(ChestMenuUtils.getNotResearchedItem(), ChatColor.WHITE + ItemUtils.getItemName(sfitem.getItem()), "&4&l" + Slimefun.getLocalization().getMessage(p, "guide.locked"), "", "&a> Click to unlock", "", "&7Cost: &b" + research.getCost() + " Level(s)"));
+            menu.addItem(index, new CustomItemStack(ChestMenuUtils.getNotResearchedItem(), ChatColor.WHITE + ItemUtils.getItemName(sfitem.getItem()), "&4&l" + Slimefun.getLocalization().getMessage(p, "guide.locked"), "", "&a>" + Slimefun.getLocalization().getMessage(p, "guide.click-unlock"), "", "&7" + Slimefun.getLocalization().getMessage(p, "guide.Cost") + research.getCost() + Slimefun.getLocalization().getMessage(p, "guide.Level")));
```

Comment:
the key should be in lowercase, same like the `messages.yml`
```suggestion
            menu.addItem(index, new CustomItemStack(ChestMenuUtils.getNotResearchedItem(), ChatColor.WHITE + ItemUtils.getItemName(sfitem.getItem()), "&4&l" + Slimefun.getLocalization().getMessage(p, "guide.locked"), "", "&a>" + Slimefun.getLocalization().getMessage(p, "guide.click-unlock"), "", "&7" + Slimefun.getLocalization().getMessage(p, "guide.cost") + research.getCost() + Slimefun.getLocalization().getMessage(p, "guide.level")));
```